auth_block_signup/models/res_users.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
No need to revive.

Reply:
I reformat with pre-commit again and push new commit. Those lines are removed.